wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -411,14 +412,9 @@ class PlatformRepo @Inject constructor(
     fun observeContacts(text: String, orderBy: UsernameSortOrderBy, includeSentPending: Boolean = false): Flow<List<UsernameSearchResult>> {
         return blockchainIdentityDataStorage.observe()
             .filterNotNull()
-            .flatMapLatest { _ ->
-                init()
-
-                if (!hasIdentity) {
-                    return@flatMapLatest flowOf(emptyList())
-                }
-
-                val userId = blockchainIdentity.uniqueIdString
+            .filter { it.creationState >= BlockchainIdentityData.CreationState.DONE }
+            .flatMapLatest { identityData ->
+                val userId = identityData.userId!!
```

Comment:
Found a crash due during wallet restoration.  Let's make sure that restoration of the identity is finished first.